scene/3d/xr_hand_modifier_3d.cpp

```diff
@@ -176,74 +184,67 @@ void XRHandModifier3D::_process_modification() {
 		return;
 	}
 
-	XRServer *xr_server = XRServer::get_singleton();
+	const XRServer *xr_server = XRServer::get_singleton();
 	if (!xr_server) {
 		return;
 	}
 
-	Ref<XRHandTracker> tracker = xr_server->get_hand_tracker(tracker_name);
-	if (tracker.is_null()) {
+	const Ref<XRHandTracker> tracker = xr_server->get_tracker(tracker_name);
+	if (!tracker.is_valid()) {
+		return;
+	}
+
+	// Skip if no tracking data
+	if (!tracker->get_has_tracking_data()) {
 		return;
 	}
 
 	// Get the world and skeleton scale.
-	const float ws = xr_server->get_world_scale();
```

Comment:
This code would apply the world_scale to the XXHandModifier3D transform scaling it (and all child nodes such as meshes).

Now that position is driven by XRNode3D we get the same functionality because:
* `XRNode3D::_pose_changed` sets the node transform from `XRPose::get_adjusted_transform`
* `XRPose::get_adjusted_transform` scales the transform by the world_scale

Reply:
Ah, ok, great! I had incorrectly thought this was being removed per our earlier discussions around deprecating/removing world scale. Anyway, I'm glad to hear that it's still being applied, just in a different place.